<?xml version="1.0"?>
<launch>
    
  <node name="jetson_system_monitor" pkg="puma_system_monitor" type="system_monitor_node.py" output="log" respawn="true" respawn_delay="1" clear_params="true">
    <param name="thermal_zone" value="1"/>
    <remap from="/cpu_usage" to="/puma/jetson/cpu_usage"/>
    <remap from="/cpu_temperature" to="/puma/jetson/cpu_temperature"/>
    <remap from="/memory_usage" to="/puma/jetson/memory_usage"/>
  </node>

  <node name="jetson_services_manager" pkg="puma_system_monitor" type="services_manager_node.py" output="log" respawn="true" respawn_delay="2" clear_params="true">
    <param name="ns_service" value="puma/jetson"/>
  </node>

</launch>